core/src/View/Kokkos_ViewTraits.hpp

```diff
@@ -455,12 +459,12 @@ struct ViewTraits {
   //------------------------------------
   // Execution space, memory space, memory access traits, and host mirror space.
 
-  using execution_space   = ExecutionSpace;
-  using memory_space      = MemorySpace;
-  using device_type       = Kokkos::Device<ExecutionSpace, MemorySpace>;
-  using memory_traits     = MemoryTraits;
-  using host_mirror_space = HostMirrorSpace;
-  using hooks_policy      = HooksPolicy;
+  using execution_space    = ExecutionSpace;
+  using memory_space       = MemorySpace;
+  using device_type        = Kokkos::Device<ExecutionSpace, MemorySpace>;
+  using host_mirror_device = HostMirrorDevice;
+  using memory_traits      = MemoryTraits;
+  using hooks_policy       = HooksPolicy;
```

Comment:
HostMirrorSpace was a public typedef inside public ViewTraits, so it still needs to be here (in deprecated form).

Reply:
Fixed